cypress/integration/demo/trade.js

```diff
@@ -0,0 +1,32 @@
+import { Given, When, Then, And} from 'cypress-cucumber-preprocessor/steps'
+
+
+
+Given('Reactive Trader is open', () => {
+
+    //cy.visit("https://web.prod.reactivetrader.com/")
+    cy.visit("http://localhost:1917")
```

Comment:
I think we need to make this a parameter, so cy.visit(base_url") and then define base_url either here in the steps or better in config